src/routes/(settings)/settings/account/+page.svelte

```
@@ -0,0 +1,30 @@
+<script lang="ts">
+  import { createCurrentUser } from "$lib/client/hooks/useCurrentUser.svelte";
+  import { ArrowRightIcon, Loader2Icon } from "@lucide/svelte";
+  import { Card, CardContent, CardHeader, CardTitle } from "$lib/components/ui/card";
```

ESLint

[error] 'Card' is defined but never used.

(@typescript-eslint/no-unused-vars)